cmd/template-resolver/utils/resolver_client.go

```diff
@@ -10,14 +10,15 @@ import (
 
 // Struct representing the template-resolver command
 type TemplateResolver struct {
-	hubKubeConfigPath string
-	clusterName       string
-	hubNamespace      string
-	objNamespace      string
-	objName           string
-	saveResources     string
-	// saveHubResources Output doesn't include "ManagedClusters" resources,
-	saveHubResources string
+	HubKubeConfigPath string `yaml:"hubKubeConfigPath"`
+	ClusterName       string `yaml:"clusterName"`
+	HubNamespace      string `yaml:"hubNamespace"`
+	ObjNamespace      string `yaml:"objNamespace"`
+	ObjName           string `yaml:"objName"`
+	SaveResources     string `yaml:"saveResources"`
+	// saveHubResources Output doesn't include "ManagedClusters" resources
+	SaveHubResources     string `yaml:"saveHubResources"`
+	SkipPolicyValidation bool   `yaml:"skipPolicyValidation"`
```

Comment:
I didn't see a reason these weren't exported previously. I'm open to ideas to preserve the from being exported, but I don't see the harm. (The goal in exporting them is to use the struct in the tests.)